file.go

```diff
@@ -114,30 +117,31 @@ func extractFiles(zipFilePath, filesToExtractFromZipPath, localPath string) erro
 				path := filepath.Join(localPath, strings.TrimPrefix(f.Name, pathPrefix))
 				err = os.MkdirAll(path, 0777)
 				if err != nil {
-					return fmt.Errorf("Failed to create local directory %s: %s", path, err)
+					return 0, fmt.Errorf("Failed to create local directory %s: %s", path, err)
```

Comment:
NIT: I suspect that you want to return `fileCount` in all these cases, as it's possible that the error happened after, say, 5 files were extracted, so returning 0 wouldn't be accurate.

Reply:
Agreed and changed.